ui-src/package.json

```diff
@@ -37,7 +38,7 @@
     "zone.js": "^0.8.14"
   },
   "devDependencies": {
-    "@angular/cli": "1.6.8",
+    "@angular/cli": "1.6.3",
```

Comment:
Why going down a version? Just curious

Reply:
@AlexeiSuleac as this was the origanal version we started with. It doesn't really make a difference any way it is only the patch version and should be backward compatible.